sql/firefox_desktop_exact_mau28_v1.sql

```diff
@@ -1,9 +1,11 @@
 SELECT
   submission_date,
-  CURRENT_DATETIME() AS generated_time,
   SUM(mau) AS mau,
   SUM(wau) AS wau,
-  SUM(dau) AS dau
+  SUM(dau) AS dau,
+  SUM(active_mau) AS active_mau,
+  SUM(active_wau) AS active_wau,
+  SUM(active_dau) AS active_dau
```

Comment:
Do you feel like `firefox_desktop_exact_mau28_by_dimensions_v1` is small enough that this table should become a live view on top of it rather than a materialized view?

Reply:
good point, that table is `33,809,840,279` bytes right now, so querying the whole thing costs ~17 cents. live view seems good, and then we don't need backfill.